lib/logstash/filters/cipher.rb

```diff
@@ -73,17 +74,58 @@ class LogStash::Filters::Cipher < LogStash::Filters::Base
   # If you want to change it, set this parameter. If you want to disable
   # it, Set this parameter to 0
   # [source,ruby]
-  #     filter { cipher { padding => 0 }}
+  #     filter { cipher { cipher_padding => 0 }}
   config :cipher_padding, :validate => :string
 
-  # The initialization vector to use
+  # The initialization vector to use (statically hard-coded). For
+  # a random IV see the iv_random_length property
+  #
+  # NOTE: If iv_random_length is set, it takes precedence over any value set for "iv"
   #
   # The cipher modes CBC, CFB, OFB and CTR all need an "initialization
   # vector", or short, IV. ECB mode is the only mode that does not require
   # an IV, but there is almost no legitimate use case for this mode
   # because of the fact that it does not sufficiently hide plaintext patterns.
+  #
+  # For AES algorithms set this to a 16 byte string.
+  # [source,ruby]
+  #     filter { cipher { iv => "1234567890123456" }}
+  #
+  # Deprecated: Please use `iv_random_length` instead
```

Comment:
This should get a `:deprecated => "Please use 'iv_random_length'"` option passed in.